src/components/partner/dto/create-partner.dto.ts

```diff
@@ -1,8 +1,9 @@
-import { Field, InputType, ObjectType } from '@nestjs/graphql';
+import { Field, ID, InputType, ObjectType } from '@nestjs/graphql';
 import { Transform, Type } from 'class-transformer';
 import { Matches, ValidateNested } from 'class-validator';
 import { uniq } from 'lodash';
-import { ID, IdField, IdOf, NameField } from '../../../common';
+import { ID as CordID, IdField, IdOf, NameField } from '../../../common';
```

Comment:
Let's keep this as the simple `ID` name.

```suggestion
import { ID, IdField, IdOf, NameField } from '../../../common';
```